solutions/cses-1745.mdx

```diff
@@ -2,20 +2,84 @@
 id: cses-1745
 source: CSES
 title: Money Sums
-author: Sofia Yang
+author: Sofia Yang, Qian Qian
 ---
 
 [Official Editorial](https://codeforces.com/blog/entry/70018)
 
+## Solution
+
+**Time Complexity:** $\mathcal{O}(N\cdot X)$
+
+## Implementation
+
 <LanguageSection>
 
-<JavaSection>
+<CPPSection>
 
-## Solution
+``` cpp
+#include <bits/stdc++.h>
 
-**Time Complexity:** $\mathcal{O}(N\cdot X)$
+using namespace std;
 
-## Java Implementation
+using vi = vector<int>;
```

Comment:
don't think this is in the allowed macro list?

Reply:
This is written in the part [Type Aliases With `using`](https://usaco.guide/general/generic-code?lang=cpp#type-aliases-with-using) of USACO Guide, so I just used it, but remove it and use `vector<int>` in other parts shouldn't be a problem.

Reply:
address this & we're done

Reply:
> address this & we're done

done